src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -276,7 +277,10 @@ private void initKeyBindings() {
                         break;
                     case PASTE:
                         if (OS.OS_X) { // Workaround for a jdk issue that executes paste twice when using cmd+v in a TextField
-                            event.consume();
+                            // Extra workaround for CodeArea, which does not inherit from TextInputControl
+                            if (!(stateManager.getFocusOwner().isPresent() && (stateManager.getFocusOwner().get() instanceof CodeArea))) {
+                                event.consume();
+                            }
                             break;
```

Comment:
Did not see this earlier: Should the break statement be part of the if-mac-clause?

Reply:
It's part of the outer mac clause.